certora/specs/Health.spec

```diff
@@ -0,0 +1,115 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+import "ConsistentInstantiation.spec";
+import "SummaryLib.spec";
+
+methods {
+    function _.onMorphoRepay(uint256, bytes) external => DISPATCHER(true);
+
+    // Disregard some unresolved calls altogether.
+    function _.onPreLiquidate(uint256, bytes) external => NONDET DELETE;
+    function _.transfer(address, uint256) external => NONDET DELETE;
+    function _.transferFrom(address, address, uint256) external => NONDET DELETE;
+
+    function _.price() external => constantPrice expect uint256;
+
+    function marketParams() internal returns (PreLiquidation.MarketParams memory)
+        => summaryMarketParams();
+
```

Comment:
Why do we need this summary ? The function marketParams() is doing the same thing as the summary

Reply:
Re-opening because we can remove summaryMarketParams() entirely I think